packages/rest/src/rest-server.ts

```diff
@@ -98,7 +102,12 @@ export class RestServer extends Context implements Server {
   }
   protected _httpServer: Http.Server;
 
-  constructor(app: Context, public options?: RestServerOptions) {
+  constructor(
+    @inject(CoreBindings.APPLICATION_INSTANCE)
+    app?: Application,
+    @inject(CoreBindings.APPLICATION_CONFIG)
+    options?: RestServerOptions,
```

Comment:
This will inject the full app config, not the rest server options only. I think we need to change `@inject` key to `CoreBindings.APPLICATION_CONFIG + '#rest')`.

A use case to consider (possibly later, outside of this pull request): how to handle the situation when there are multiple RestServer instances registered with the app, each given their own server name (e.g. `v1api`)? Should we use a different key to retrieve the server config from the app config?